go.mod

```diff
@@ -1,6 +1,19 @@
 module github.com/containers/podman/v5
 
-go 1.20
+// Minimum required golang version
+go 1.20 // *****  ATTENTION  WARNING  CAUTION  DANGER  ******
+
+//         Go versions 1.21 and later will AUTO-UPDATE based
+//         on _module_requirements_.
+//         ref: https://go.dev/doc/toolchain  Because several
+//         different distros and distro-versions build from
+//         this code, golang version consistency is
+//         desireable.  After manually updating to 1.21, a
+//         `toolchain` specificication should be added to pin
+//         the version and block auto-updates.
```

Comment:
This sentence is talking about `toolchain` (and is correct about that); it does not block `go`.